File: Java/bee 1006.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double A, B, C;
        A = sc.nextDouble() * 2;
        B = sc.nextDouble() * 3;
        C = sc.nextDouble() * 5;
        double MEDIA = (A + B + C)/10;

        System.out.printf("MEDIA = %.1f\n", MEDIA);
    }
}

File: Kotlin/bee 1006.kt
import java.util.*
import java.io.BufferedReader
import java.io.InputStreamReader

fun main(args: Array<String>) {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val a = br.readLine().toDouble() * 2
    val b = br.readLine().toDouble() * 3
    val c = br.readLine().toDouble() * 5

    println(String.format("MEDIA = %.1f", (a+b+c)/10))
}
